packages/orbit-components/src/List/index.tsx

```diff
@@ -1,76 +1,54 @@
 import * as React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import type * as Common from "../common/types";
-import type { Theme } from "../defaultTheme";
-import defaultTheme from "../defaultTheme";
 import { SIZES, TYPES } from "./consts";
-import getSpacingToken from "../common/getSpacingToken";
-import { getLineHeightToken } from "./ListItem";
-import ListContext from "./ListContext";
-import type { Type, Props } from "./types";
-
-const getSizeToken = ({ theme, size }: { theme: Theme; size?: Common.Size }): string | null => {
-  const sizeTokens = {
-    [SIZES.SMALL]: theme.orbit.fontSizeTextSmall,
-    [SIZES.NORMAL]: theme.orbit.fontSizeTextNormal,
-    [SIZES.LARGE]: theme.orbit.fontSizeTextLarge,
-  };
-
-  if (!size) return null;
-
-  return sizeTokens[size];
+import type { Props } from "./types";
+import SPACINGS_AFTER from "../common/getSpacingToken/consts";
+
+const spaceAfterTokens = {
```

Comment:
We already have these tokens-classes match in a common file. Under `src/common/tailwind/`